core/templates/pages/exploration-editor-page/feedback-tab/feedback-tab.component.html

```diff
@@ -11,75 +11,77 @@ <h2>Exploration Feedback</h2>
       </div>
     </div>
 
-    <div class="row" ng-if="$ctrl.activeThread.messages">
+    <div class="row" *ngIf="activeThread.messages">
       <div class="col-lg-12 col-md-12 col-sm-12 activeThreadView">
         <table class="table">
-          <tr ng-repeat="m in $ctrl.activeThread.messages|orderBy:'messageId'">
+          <tr *ngFor="let m of activeThread.messages; index as index">
             <td>
               <div class="row">
                 <div class="col-lg-5 col-md-5 col-sm-5">
-                  <span ng-if="$index !== 0">#<[m.messageId]></span>
-                  <span ng-if="m.authorUsername">by <strong><[m.authorUsername]></strong></span>
-                  <span ng-if="!m.authorUsername">(anonymously submitted)</span>
+                  <span *ngIf="index !== 0">#{{ m.messageId }}</span>
+                  <span *ngIf="m.authorUsername">by <strong>{{ m.authorUsername }}</strong></span>
+                  <span *ngIf="!m.authorUsername">(anonymously submitted)</span>
                 </div>
 
                 <div class="col-lg-4 col-md-4 col-sm-4">
-                  <span ng-if="m.messageId !== 0">
-                    <span ng-if="m.updatedStatus">
-                      <em>Status changed to '<[$ctrl.getHumanReadableStatus(m.updatedStatus)]>'</em>
+                  <span *ngIf="m.messageId !== 0">
+                    <span *ngIf="m.updatedStatus">
+                      <em>Status changed to '{{ getHumanReadableStatus(m.updatedStatus) }}'</em>
                     </span>
-                    <span ng-if="m.updatedSubject">
-                      <em>Subject changed to '<[m.updatedSubject]>'</em>
+                    <span *ngIf="m.updatedSubject">
+                      <em>Subject changed to '{{ m.updatedSubject }}'</em>
                     </span>
                   </span>
                 </div>
 
                 <div class="col-lg-3 col-md-3 col-sm-3">
-                  <span><[$ctrl.getLocaleAbbreviatedDatetimeString(m.createdOnMsecs)]></span>
+                  <span>{{ getLocaleAbbreviatedDatetimeString(m.createdOnMsecs) }}</span>
                 </div>
               </div>
 
               <div class="row">
                 <div class="col-lg-12 col-md-12 col-sm-12">
-                  <div class="oppia-view-suggestion-button-container e2e-test-exploration-feedback"><[m.text]></div>
-                  <div ng-if="$ctrl.activeThread.isSuggestionThread() && $index == 0">
-                    <button class="btn btn-<[$ctrl.getSuggestionButtonType()]> e2e-test-view-suggestion-btn oppia-view-suggestion-btn" ng-click="$ctrl.showSuggestionModal()">
+                  <div class="oppia-view-suggestion-button-container e2e-test-exploration-feedback">{{ m.text }}</div>
+                  <div *ngIf="activeThread.isSuggestionThread() && index == 0">
+                    <button class="btn btn-{{ getSuggestionButtonType() }} e2e-test-view-suggestion-btn oppia-view-suggestion-btn" (click)="showSuggestionModal()">
```

Comment:
```suggestion
                    <button class="btn btn-{{ getSuggestionButtonType() }} e2e-test-view-suggestion-btn oppia-view-suggestion-btn" 
                            (click)="showSuggestionModal()">
```

Reply:
Please try to do similar changes elsewhere, even though we do not follow the 80-line length limit strictly in the HTML files, it is still nice to make the lines shorter.